openy.profile

```diff
@@ -635,6 +708,12 @@ function openy_install_tasks_alter(&$tasks, &$install_state) {
     isset($tasks["openy_third_party_services"])) {
       unset($tasks["openy_third_party_services"]);
   }
+  // Remove Solr configure installation task for non search_api sorl service.
+  if (!empty(\Drupal::state()->get('openy_hide_solr_config')) &&
```

Comment:
Can we simplify this condition?
I think `if (\Drupal::state()->get('openy_hide_solr_config')) {}` does the same job here

Reply:
Also, I think we can use the opposite condition here. See comment about in `openy_install_search()`. Something like 
```
if (!\Drupal::state()->get('openy_show_solr_config')) {
    unset($tasks["openy_solr_search"]);
}
```